contracts/token/ERC721/extensions/ERC721URIStorage.sol

```diff
@@ -35,6 +35,18 @@ abstract contract ERC721URIStorage is ERC721 {
         return super.tokenURI(tokenId);
     }
 
+    /**
+     * @dev Gets the tokenURI of `tokenId`.
+     *
+     * Requirements:
+     *
+     * - `tokenId` must exist.
+     */
+    function _getTokenURI(uint256 tokenId) internal view virtual returns (string memory) {
```

Comment:
This function needs another name that makes it clearer that this is different from `tokenURI`.